src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.File.cs

```diff
@@ -56,7 +56,7 @@ public string Sdk
         public bool IsSdk =>
             ProjectRoot.Sdk is not null && ProjectRoot.Sdk.Contains(MSBuildConstants.DefaultSDK, StringComparison.OrdinalIgnoreCase);
 
-        public IEnumerable<string> Imports => ProjectRoot.Imports.Select(p => Path.GetFileName(p.Project));
+        public ICollection<string> Imports => ProjectRoot.Imports.Select(p => Path.GetFileName(p.Project)).ToList();
```

Comment:
As mentioned above, this can probably stay an enumerable.

Reply:
Removed the Add/Remove methods and added a custom collection to support the add/remove operation